### ISSUE TITLE
Modifying multus-daemonset.yml is invalid

### ISSUE THREAD

**Issue by jiayoukun** [k8snetworkplumbingwg/multus-cni#1110]
**What happend**:
The current problems are: When I modified flannel into calico from plugins in configmap of multus-daemonset.yml and executed kubectl apply command again, multus does not modify the /etc/cni/net.d/00-multus.conf file; it will still be invoked using the flannel CNI.
After I deleted /etc/cni.net.d/00-multus. conf, I re-executed the kubectl apply multus-daemonset.yml file to correctly use it.

I think in code cmd/multus-daemon/main.go
MultusConf. MultusConfigFile USES to identify whether there is wrong?
Waiting for your reply, thank you!

**Environment**:

- Multus version v4.0.x
  image path and image ID (from 'docker images')
- Kubernetes version (use `kubectl version`): v1.24.10
- Primary CNI for Kubernetes cluster: calico or flannel
- OS (e.g. from /etc/os-release): CentOS 8
- File of '/etc/cni/net.d/'

**Comment by nicklesimba** [comment]
Hi, as a first step, have you changed the `multus-conf-file` flag? See: [Using the configmap in the multus-daemonset ](https://github.com/k8snetworkplumbingwg/multus-cni/blob/master/deployments/multus-daemonset.yml#L111-L118)

**Comment by jiayoukun** [comment]
@nicklesimba 
I'm not trying to manually modify the configuration, I'm just stating that modifying the multus-daemonset.yml file in this scenario will not work!

In my scenario, the first time I finished adding the multus-daemonset.yml file, the 00-multus.conf file was generated under the /etc/cni/net. d/ path, and after using it for a while, I needed to modify the cni type of the 00-multus.conf file, so I reworked and added the multus- daemonset.yml file, but the code recognized that the 00-multus.conf file already existed and would not be modified for 00-multus.conf. Using its contents directly would invalidate the changes to the multus-daeminit.yml file.
